.travis.yml

```diff
@@ -8,8 +8,6 @@ env:
   global:
     - DOCKER_COMPOSE_VERSION=1.20.1
       # get all the branches referencing this commit
-    - IS_RELEASE=$(git ls-remote origin | grep "$TRAVIS_COMMIT\s\+refs/heads/release$")
```

Comment:
There are no references to this variable later in the file. Was it just a leftover crumb, or does Travis use it in some other way?

Reply:
this was a left over variable that had no references in all of codebases, it precedes me so I'm not sure how it was used.

Actually, I found the changes came from Yotam for [AC-5347](https://masschallenge.atlassian.net/browse/AC-5347).

The changes go further back to a change by Shanks in this [PR](https://github.com/masschallenge/impact-api/pull/114/files)
We stopped using the `IS_RELEASE` variable and didn't get rid of it.